src/Storefront/Resources/views/storefront/component/form/form-input.html.twig

```diff
@@ -78,7 +78,7 @@
                    {% if maxlength is not empty %}maxlength="{{ maxlength }}"{% endif %}
                    {% if validationRules is not empty %}data-validation="{{ validationRules }}"{% endif %}
                    {% if 'required' in validationRules %}aria-required="true"{% endif %}
-                   {% if attributes is not empty %}{% for key, value in attributes %}{% if value is not empty %}{{ key ~ '=' ~ value ~ ' ' }}{% endif %}{% endfor %}{% endif %}
+                   {% if attributes is not empty %}{% for key, value in attributes %}{% if value is not empty %}{{ key }}="{{ value }}"{% else %}{{ key }}{% endif %}{% endfor %}{% endif %}
```

Comment:
You forgot the empty space afterwards, if "attributes" has multiple ones, they'll be put toghter like `attribute1="foo"attribute2=bar` and so on.